code/game/gamemodes/objective.dm

```diff
@@ -314,16 +358,16 @@
 
 	return 0
 
-/datum/objective/escape/escape_with_identity
+/datum/objective/escape_obj/escape/escape_with_identity

```

Comment:
Wouldn't a normal crewmember be able to pull it off though? Stealing the guy's identity via genetics and ID.